source/MRMesh/MRObjectLoad.cpp

```diff
@@ -130,6 +131,7 @@ Expected<ObjectGcode, std::string> makeObjectGcodeFromFile( const std::filesyste
 
     ObjectGcode objectGcode;
     objectGcode.setName( utf8string( file.stem() ) );
+    objectGcode.setCNCMachineSettings( SceneSettings::getCNCMachineSettings() );
```

Comment:
I think this should be in ObjectGcode constructor (it is default parameters so we dont need to setup it manualy)